packages/tasks/package.json

```diff
@@ -7,33 +7,36 @@
 	"publishConfig": {
 		"access": "public"
 	},
-	"main": "./dist/index.cjs",
-	"module": "./dist/index.js",
-	"types": "./dist/index.d.ts",
+	"sources": "./src/index.ts",
+	"main": "./dist/commonjs/index.js",
+	"module": "./dist/esm/index.js",
+	"types": "./dist/commonjs/index.d.ts",
 	"exports": {
+		"./package.json": "./package.json",
 		".": {
-			"require": "./dist/index.cjs",
-			"import": "./dist/index.js"
+			"import": {
+				"types": "./dist/esm/index.d.ts",
+				"default": "./dist/esm/index.js"
+			},
+			"require": {
+				"types": "./dist/commonjs/index.d.ts",
+				"default": "./dist/commonjs/index.js"
+			}
 		}
 	},
```

Comment:
that's the result of having run `tshy` locally?

Reply:
yes, it did that itself